src/lib/format.js

```diff
@@ -23,18 +23,24 @@ export function formatAddress(address) {
 }
 
 const statusColourMap = {
-  Registered: "green",
+  Cancelled: "red",
+  "Cannot register": "red",
   Draft: "purple",
+  "De-registered": "red",
+  Deleted: "red",
+  "Do not register": "red",
+  Expired: "red",
+  "In progress": "light-blue",
   Perfect: "turquoise",
-  Pending: "yellow",
+  Pending: "blue",
```

Comment:
Test seems to be failing on blue being yellow?

Reply:
Ah yep, I've updated that now 👍🏽